_assets/ci/Jenkinsfile-status-go

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env groovy
+
+parameters {
+  string(
+    name: 'DOCKER_TAG',
+    defaultValue: "deploy-staging",
+    description: 'Under what tag to push the docker image.'
+  )
+}
+
+properties([disableConcurrentBuilds()])
+
+node('linux') {
+
+  def image_name = 'statusteam/status-go'
+  def commit image build_flags
+  def tstamp = Calendar.getInstance().getTime().format(
+    'YYYYMMdd-hhmmss',TimeZone.getTimeZone('UTC')
+  )
+
+  stage('Git Prep') {
+    checkout scm
+    commit = sh(
+      returnStdout: true,
+      script: 'git rev-parse --short HEAD'
+    ).trim()
+    build_flags = "-ldflags '-X main.buildStamp=${tstamp}\
+    build_flags += "-X github.com/status-im/status-go/params.VersionMeta=${commit}'"
+  }
+
+  stage('Build') {
+    image = docker.build(
+      image_name + ':' + commit,
+      "--build-arg git_commit=${commit} " +
```

Comment:
I guess the only thing left to do is to drop this line and instead add:
```
"--label 'commit=${commit}'" +
"--label 'author=jenkins'" +
```
And call it a day, ALTHOUGH, this builds only the bootnode. I don't know if we want separate builds, or make both as part of the same build?

Reply:
The jenkinsfile will be almost identical for both as far as I can tell.

Reply:
Yeah, so that was surprising for me that it only builds bootnode. What's the big picture behind this file? You just wanted to have a way to build Docker images from Jenkins?

Reply:
sure, remove it